Sources/Orbit/Components/Slider.swift

```diff
@@ -0,0 +1,441 @@
+
+                        guard newHorizontalPosition > 0 else {
+                            onValueChange(bounds.lowerBound)
+                            return
+                        }
+
+                        if buckets.count > 1 {
+                            let currentValue = currentValue(location: newHorizontalPosition, width: width)
+                            guard let bucket = buckets.first(where: { $0.contains(currentValue) }) else { return }
+                            onValueChange(currentValue < bucket.middle ? bucket.lowerBound : bucket.upperBound)
+                        } else {
+                            onValueChange(currentValue(location: newHorizontalPosition, width: width))
+                        }
+                    }
+                    .onEnded { _ in
+                        isDragging.wrappedValue = false
+                        dragStartOffset = 0
+                    }
+            )
+    }
+
+    @ViewBuilder func strokedPath(from: CGFloat, to: CGFloat) -> some View {
+        Path { path in
+            path.move(to: .init(x: from, y: lineCenter))
+            path.addLine(to: .init(x: to, y: lineCenter))
+        }
+        .stroke(Color.blueNormal, lineWidth: lineHeight)
+    }
+
+    private var lineHeight: CGFloat {
+        CGFloat.xxSmall * sizeCategory.controlRatio
+    }
+
+    private var lineCenter: CGFloat {
+        lineHeight / 2
+    }
+
+    private var thumbVerticalOverlapping: CGFloat {
+        (SliderThumb.circleDiameter / 2 - lineCenter) * sizeCategory.controlRatio
+    }
+
+    private var accessibilityHint: String {
+        "\(startLabel) - \(endLabel)"
+    }
+
+    private func updateSelectedRangeIfNeeded() {
+        if let upperBound = secondValue, case .range(let selectedRange) = value {
+            selectedRange?.wrappedValue = min(firstValue, upperBound)...max(firstValue, upperBound)
+        }
+    }
+
+    private func currentValue(location: CGFloat, width: CGFloat) -> Value {
+        bounds.lowerBound + Value(location / width) * bounds.distance
+    }
+
+    private func position(by value: Value, width: CGFloat) -> CGFloat {
+        let position = CGFloat((value - bounds.lowerBound) / bounds.distance) * width
+        return min(width, max(0, position))
+    }
+
+    public init(
+        value: Binding<Value>,
+        valueLabel: String,
+        startLabel: String,
+        endLabel: String,
+        bounds: ClosedRange<Value>,
+        step: Value.Stride? = nil
+    ) {
+        assert(step != 0, "step can't be 0")
+
+        self.value = .single(value)
+        self.valueLabel = valueLabel
+        self.startLabel = startLabel
+        self.endLabel = endLabel
+
+        self.bounds = bounds
+
+        self._firstValue = State(initialValue: value.wrappedValue)
+        self.buckets = bounds.chunked(into: step ?? Value.Stride(bounds.upperBound - bounds.lowerBound))
+    }
+
+    public init(
+        selectedRange: Binding<ClosedRange<Value>>,
```

Comment:
Suggestion - to match the other init, this can be named `values` or `valueRange`